keep/providers/uptimekuma_provider/uptimekuma_provider.py

```diff
@@ -48,7 +48,7 @@ class UptimekumaProviderAuthConfig:
     )
 
 
-class UptimekumaProvider(BaseProvider):
+class UptimeKumaProvider(BaseProvider):
```

Comment:
@pehlicd i'm not sure if this will work or not. we need it to be lowercase, is it critical?